src/Company/Company.php

```diff
@@ -296,4 +302,12 @@ public function getBic(): string
     {
         return $this->bic;
     }
+
+    /**
+     * @return string
+     */
+    public function getNafCode(): string
```

Comment:
Cohérence :stuck_out_tongue: 

```suggestion
    public function getNafCode(): ?string
```